src/sentry/testutils/factories.py

```diff
@@ -154,6 +154,7 @@
 from sentry.types.actor import Actor
 from sentry.types.region import Region, get_local_region, get_region_by_name
 from sentry.types.token import AuthTokenType
+from sentry.uptime import models as uptime_models
```

Comment:
Feels weird importing this like this since we already import specific models right below.

maybe just alias them instead?

```
Region as UptimeRegion
```

or something like that?